libs/langchain/langchain/document_loaders/parsers/audio.py

```diff
@@ -219,3 +219,57 @@ def lazy_parse(self, blob: Blob) -> Iterator[Document]:
             page_content=prediction,
             metadata={"source": blob.source},
         )
+
+
+class YandexSTTParser(BaseBlobParser):
+    """Transcribe and parse audio files.
+    Audio transcription is with OpenAI Whisper model."""
+
+    def __init__(
+        self,
```

Comment:
Use `*` operator if including defaults for named arguments

```suggestion
        self,
        *,
```